src/tools/cursors/index.js

```diff
@@ -0,0 +1,384 @@
+      39-17 69 14l144 144 355-355-355-355-144 144q-19 19-45 19-12
+      0-24-5-40-17-40-59v-448q0-26 19-45t45-19h448q42 0 59 40 17 39-14 69l-144
+      144 355 355 355-355-144-144q-31-30-14-69 17-40 59-40h448q26 0 45 19t19
+      45v448q0 42-39 59-13 5-25 5-26 0-45-19z"
+    />`,
+  {
+    viewBox: {
+      x: 1792,
+      y: 1792,
+    },
+  }
+);
+
+export const rotateCursor = new MouseCursor(
+  `<path fill="ACTIVE_COLOR" d="M1664 256v448q0 26-19 45t-45 19h-448q-42 0-59-40-17-39
+      14-69l138-138q-148-137-349-137-104 0-198.5 40.5t-163.5 109.5-109.5
+      163.5-40.5 198.5 40.5 198.5 109.5 163.5 163.5 109.5 198.5 40.5q119 0
+      225-52t179-147q7-10 23-12 15 0 25 9l137 138q9 8 9.5 20.5t-7.5 22.5q-109
+      132-264 204.5t-327 72.5q-156 0-298-61t-245-164-164-245-61-298 61-298
+      164-245 245-164 298-61q147 0 284.5 55.5t244.5 156.5l130-129q29-31 70-14
+      39 17 39 59z"
+    />`,
+  {
+    viewBox: {
+      x: 1792,
+      y: 1792,
+    },
+  }
+);
+
+export const stackScrollCursor = new MouseCursor(
+  `<path fill="ACTIVE_COLOR" d="M24 21v2c0 0.547-0.453 1-1 1h-22c-0.547
+      0-1-0.453-1-1v-2c0-0.547 0.453-1 1-1h22c0.547 0 1 0.453 1 1zM24 13v2c0
+      0.547-0.453 1-1 1h-22c-0.547 0-1-0.453-1-1v-2c0-0.547 0.453-1 1-1h22c0.547
+      0 1 0.453 1 1zM24 5v2c0 0.547-0.453 1-1 1h-22c-0.547
+      0-1-0.453-1-1v-2c0-0.547 0.453-1 1-1h22c0.547 0 1 0.453 1 1z"
+      />`,
+  {
+    viewBox: {
+      x: 24,
+      y: 28,
+    },
+  }
+);
+
+export const wwwcRegionCursor = new MouseCursor(
+  `<path fill="ACTIVE_COLOR" d="M1664 416v960q0 119-84.5 203.5t-203.5 84.5h-960q-119
+    0-203.5-84.5t-84.5-203.5v-960q0-119 84.5-203.5t203.5-84.5h960q119 0 203.5
+    84.5t84.5 203.5z"
+  />`,
+  {
+    viewBox: {
+      x: 1792,
+      y: 1792,
+    },
+  }
+);
+
+export const wwwcCursor = new MouseCursor(
+  `<path fill="ACTIVE_COLOR" d="M14.5,3.5 a1 1 0 0 1 -11,11 Z" stroke="none" opacity="0.8" />
+    <circle cx="9" cy="9" r="8" fill="none" stroke-width="2" stroke="ACTIVE_COLOR" />`,
+  {
+    viewBox: {
+      x: 18,
+      y: 18,
+    },
+  }
+);
+
+export const zoomCursor = new MouseCursor(
+  `<path fill="ACTIVE_COLOR" d="M508.5 481.6l-129-129c-2.3-2.3-5.3-3.5-8.5-3.5h-10.3C395
+      312 416 262.5 416 208 416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c54.5
+      0 104-21 141.1-55.2V371c0 3.2 1.3 6.2 3.5 8.5l129 129c4.7 4.7 12.3 4.7 17
+      0l9.9-9.9c4.7-4.7 4.7-12.3 0-17zM208 384c-97.3 0-176-78.7-176-176S110.7 32 208
+      32s176 78.7 176 176-78.7 176-176 176z"
+    />
+    <path fill="ACTIVE_COLOR" transform="scale(0.22,0.22) translate(1400,0)" d="M1216
+      320q0 26-19 45t-45 19h-128v1024h128q26 0 45 19t19 45-19 45l-256 256q-19
+      19-45 19t-45-19l-256-256q-19-19-19-45t19-45 45-19h128v-1024h-128q-26
+      0-45-19t-19-45 19-45l256-256q19-19 45-19t45 19l256 256q19 19 19 45z"
+    />`,
+  {
+    viewBox: {
+      x: 640,
+      y: 512,
+    },
+  }
+);
+
+export default {
```

Comment:
Is this `export default` necessary? Every cursor is being exported individually and if a consumer of this wants every cursor they can do this:

```js
import * as cursors from './cursors';
```

Reply:
We use it in the lib to export all of the default cursors, but yeah sure, its maybe a bit superfluous. I think its only going to be used in one place, so I'll make your suggested change in the lib.

Reply:
Changed :).